sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/GuidUtilities.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Azure.Messaging.ServiceBus.Primitives;
+
+internal static class GuidUtilities
+{
+    /// <summary>
+    /// The size, in bytes, to use for extracting the delivery tag bytes into <see cref="Guid"/>.
```

Comment:
nit:  Since this is general now, we probably want to remove the reference to delivery tag here.